Docker/Dockerfile.main

```diff
@@ -2,37 +2,40 @@
 # application based on the official python image and the application code
 
 # Stage 1: Build the application
-FROM python:3.10 as build
+FROM python:3.10.15 as build
 LABEL Stage="Build"
 
 ENV HOME_DIR=/usr/src/backend
+ENV DATA_DIR=/data
 ENV PYTHONUNBUFFERED 1
 
 # Set the working directory to /usr/src/backend
 WORKDIR ${HOME_DIR}
 
+RUN mkdir -p ${DATA_DIR}
+
 COPY requirements.txt /requirements.txt
 
 # install uv. This makes installation of packages faster compared to pip
 ADD https://astral.sh/uv/install.sh /install.sh
 
 #Install system dependencies. git is needed because there are some packages that are not available in PyPI
-RUN apt-get update && apt-get -y install libpq-dev gcc wget curl git \
+RUN apt-get update && apt-get -y install libpq-dev gcc wget curl git curl \
```

Comment:
`curl` appears twice

Reply:
you are right , thanks for pointing it out.